Keystrokes to adjust applications volume and mute [take 2] (#16591)
nvaccess/nvda: user_docs/en/changes.md

### Important notes

### New Features

> * The volume of the other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. (#16052, @mltony)

Reviewer: "The" not required in "The volume of other applications"

```suggestion
* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. (#16052, @mltony)
```

> * The sound of the other applications can be muted with `NVDA+alt+delete`. (#16052, @mltony)

Reviewer: "The" not required in "The volume of other applications"

```suggestion
* The sound of other applications can be muted with `NVDA+alt+delete`. (#16052, @mltony)
```

Reviewer 2: @Qchristensen, my bad: I am the one who has asked Tony to add "the".

My reasonning was that:
* "the other applications" was meaning "all the other applications, except NVDA"
* "other applications" may mean "some other applications but not necessarily all of them"

Given your comment and suggestion, I understand that it is not correct English.

My question:
If we write:
> * The sound of other applications can be muted with `NVDA+alt+delete`. (#16052, @mltony)

Do we understand that we are talking of the sound of all applications (implied except NVDa)?
Wouldn't be understood as "some of the other applications"?

If there is no confusion possible in English, go ahead with this wording.

### Important notes

### New Features
* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. In order to use this feature, application volume adjuster needs to be enabled in Audio pane of NVDA settings. (#16052, @mltony)
* Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`.

> In order to use this feature, the application volume adjuster needs to be enabled in Audio panel of NVDA settings. (#16052, @mltony)

Reviewer 2: A detail, but the user rather speak about "category" when the developer speaks about a "panel".

```suggestion
In order to use this feature, the application volume adjuster needs to be enabled in Audio cetegory of NVDA settings. (#16052, @mltony)
```

> In order to use this feature, the application volume adjuster needs to be enabled in Audio panel of NVDA settings. (#16052, @mltony)

Reviewer 3: Adding the word "the".

```suggestion
In order to use this feature, the application volume adjuster needs to be enabled in the Audio category of NVDA settings. (#16052, @mltony)
```

### Important notes

### New Features
* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. In order to use this feature, application volume adjuster needs to be enabled in Audio pane of NVDA settings. (#16052, @mltony)

> * Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`. In order to use this feature, application volume adjuster needs to be enabled in Audio pane of NVDA settings. (#16052, @mltony)

Reviewer 4: ```suggestion
* Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`.
In order to use this feature, the application volume adjuster needs to be enabled in Audio panel of NVDA settings. (#16052, @mltony)
```

### New Features
* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. (#16052, @mltony)

> * Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`. (#16052, @mltony)

Reviewer 4: I think there needs to be a note associated to these that you must enable this functionality in settings first.
Also please move this to 2025.1
